scripts/lib/releaseNotes.ts

```diff
@@ -150,3 +151,161 @@ function addNewReleaseNoteToc(releaseNotesNode: any, newVersion: string) {
     });
   }
 }
+
+/**
+ * Sorts the release notes dictionary by the patch version
+ * in descending order, e.g. 0.25.2 then 0.25.1.
+ *
+ * Returns a dictionary with the corresponding markdown for
+ * each patch with the entries sorted by release.
+ */
+export function sortReleaseNotesVersions(markdownByPatchVersion: {
+  [id: string]: string;
+}) {
+  const markdownByPathEntries = Object.entries(markdownByPatchVersion);
+
+  const markdownByPatchSortedArray = markdownByPathEntries.sort(
+    ([version1], [version2]) => {
+      const versionPatch1 = version1.split("rc").slice(0, 1)[0];
```

Comment:
A comment would probably help to explain how this code works, including maybe an example string.